### ISSUE TITLE
Padding on Confirm and Cancel buttons does not wrap the text

### ISSUE THREAD

**Issue by FullTimeLinux** [minetest/minetest#8509]
##### Issue type

 - Bug report
 
##### Minetest version

```
0.4.17
```

##### OS / Hardware

Operating system: Fedora Linux
CPU: 64

GPU model:
OpenGL version:

##### Summary

The text of the buttons are not properly contained in the barter table menu. There is also text after the word Swindle which is unreadable due to the text improperly wrapping. 
##### Steps to reproduce

Just walk up to Shop[76] (nickthe345) above spawn and right click on the barter table on the right when you walk into his building across the street from the nap station. 

![Currency-barter_top--confirm-cancel](https://user-images.githubusercontent.com/43261056/57056081-140b5480-6c6e-11e9-93b9-25e9e31a18d3.png)

**Comment by ClobberXD** [comment]
If you just want the button size to be increased, or the overlapping text separated, then this has to be reported in the mod's issue tracker. Going by the textures, this is probably the mod you're looking for: https://content.minetest.net/packages/VanessaE/currency/

If you're requesting that the text should automatically be resized to fit into the button, then this is the right place, and that's a good idea. :)

By the way, not everyone would know where to find the shop you've described, unless you also name the server where the shop is. :)

**Comment by FullTimeLinux** [comment]
>If you're requesting that the text should automatically be resized to fit into the button, then this is the right place, and that's a good idea. :)

Yes, that is what I'm requesting. I still opened an issue on the mod page, but I don't think it'll receive attention there.

**Comment by SmallJoker** [comment]
I also tried to make the elements scale better to this limited space. However, it's apparently not possible to use different button text sizes in Irrlicht. http://irrlicht.sourceforge.net/forum/viewtopic.php?t=28820
```C++
// parseButton()
std::wstring wlabel = translate_string(utf8_to_wide(unescape_string(label)));
const core::dimension2d<u32> label_size = m_font->getDimension(wlabel.c_str());
int font_size_new = (double)m_settings->getU16("font_size") * geom.X / label_size.X;
irr::gui::IGUIFont *font = g_fontengine->getFont(font_size_new, FM_Unspecified);

// find a way to draw a button with different text sizes
```

**Comment by FullTimeLinux** [comment]
I opened an issue on the mod's source page and talked to VanessaE. She said that because the issue is fixed in the 5.0.1 release, and the 0.4 series is end of life, the fix would not be back ported. It may be okay to close this issue. What do you think?
